static/app/components/performance/vitalsAlert/types.tsx

```diff
@@ -1,8 +1,16 @@
-export interface VitalsResult {
-  FCP: number | null;
-  LCP: number | null;
-  appStartCold: number | null;
-  appStartWarm: number | null;
+import {VITALS_TYPES} from './constants';
+
+export type VitalsKey = typeof VITALS_TYPES[number];
+
+interface BaseVitalsResult {
+  [key: VitalsKey]: number;
+  appColdStartCount: number;
+  appWarmStartCount: number;
+  fcpCount: number;
+  lcpCount: number;
 }
 
-export type VitalsKey = keyof VitalsResult;
+// couldn't figure out how to do this with an interface
+export type VitalsResult = BaseVitalsResult & {
+  projectData: Array<BaseVitalsResult & {projectId: string}>;
+};
```

Comment:
Can you extend the interface like this `interface VitalsResult extends BaseVitalsResult {
    projectData: ...
    }
` maybe?

Reply:
@udameli This didn't work originally but I fixed the issue so now I can use this style, thanks!